python-checks/src/main/resources/org/sonar/l10n/py/rules/python/S5527.html

```diff
@@ -0,0 +1,68 @@
+<p>To establish a SSL/TLS connection not vulnerable to man-in-the-middle attacks, it's essential to make sure the server presents the right
```

Comment:
this file clearly has more content then here https://jira.sonarsource.com/browse/RSPEC-5530
why?

Reply:
Indeed! The RSPEC got updated during development and I forgot to update! Thanks :)